tianshou/data/collector.py

```diff
@@ -53,6 +52,28 @@ class CollectStats(CollectStatsBase):
     """Stats of the collected episode lengths."""
 
 
+def get_fresh_collect_batch_with_current_obs(
+    obs: np.ndarray,
+    info: dict | list[dict],
+) -> RolloutBatchProtocol:
+    """Empty batch with obs and info of current state, useful for adding new data."""
+    result = Batch(
+        obs={},
```

Comment:
`obs=obs` instead of mutating below

rename to `get_empty_rollout_batch`, turn `obs` and `info` params to optional.

Pls adjust the docstring to explain why one might want an empty batch with only obs and info (b/c that's what you get out of reset before making any steps, I guess)